Makefile

```diff
@@ -196,18 +198,20 @@ deploy: ## Update Kubernetes deployments in staging (jenkins)
 
 .PHONY: push-images
 push-images: ## Push Docker images to Docker Hub (jenkins)
-	docker tag storjlabs/satellite:${TAG} storjlabs/satellite:latest
-	docker push storjlabs/satellite:${TAG}
-	docker push storjlabs/satellite:${LATEST_TAG}
-	docker tag storjlabs/storagenode:${TAG} storjlabs/storagenode:latest
-	docker push storjlabs/storagenode:${TAG}
-	docker push storjlabs/storagenode:${LATEST_TAG}
-	docker tag storjlabs/uplink:${TAG} storjlabs/uplink:latest
-	docker push storjlabs/uplink:${TAG}
-	docker push storjlabs/uplink:${LATEST_TAG}
-	docker tag storjlabs/gateway:${TAG} storjlabs/gateway:latest
-	docker push storjlabs/gateway:${TAG}
-	docker push storjlabs/gateway:${LATEST_TAG}
+	docker push storjlabs/satellite:${TAG}${CUSTOMTAG}
```

Comment:
Should the satellite-ui image be pushed too?

Reply:
I don't think there is value in pushing it as it's primarily used for development purposes.